metaflow/client/trigger_live_run.py

```diff
@@ -0,0 +1,57 @@
+from metaflow.plugins.argo.argo_live_run import ArgoLiveRun
+from .live_run import LiveRun
+
+
+def trigger_live_run(
+    plugin_name: str = "Argo",  # TODO: change default to None after testing
```

Comment:
Just checking, is this TODO what you plan to resolve in this PR?

Reply:
I'm not sure about this one. 

Options:

1. We could keep `'Argo'` as the default until our users have the option to use multiple under layers. This would hide Argo from users. However, it may be good to get users in the habit of knowing which under layer they are using, especially since we may add functionality to support other under layers. Also, Metaflow would not like having 'Argo' as default, so it would make a fork
2. We could set default to `None` now, which would be best for long term b/c Metaflow will like it more.

My hunch is we should choose option 2

Reply:
The metaflow configs sounds like a good idea. I'm not very familiar with them. Should I change the TODO to something like "TODO: use metaflow configs so that AIP customers don't have to input 'Argo' as plugin_name". I would also change the default to None.

Would this work?

Reply:
Sure. The change would be to add a config var in this file https://github.com/zillow/metaflow/blob/feature/kfp/metaflow/metaflow_config.py and import the added var here.

Can you do `TODO(<JIra ticket id>)` if you want to delay that change.

Reply:
I did the plan I suggested above